src/mpris.ts

```diff
@@ -487,7 +487,7 @@ export class MPRIS extends DBusInterface {
           CanSetFullscreen: GLib.Variant.new_boolean(false),
           CanRaise: GLib.Variant.new_boolean(true),
           HasTrackList: GLib.Variant.new_boolean(false),
-          Identity: GLib.Variant.new_string("Decibels"),
+          Identity: GLib.Variant.new_string(application_id),
```

Comment:
This should not be the application ID. It should be a "friendly name" as [per the MPRIS spec](https://specifications.freedesktop.org/mpris-spec/2.2/Media_Player.html#Property:Identity):

> A friendly name to identify the media player to users.
This should usually match the name found in .desktop files
(eg: "VLC media player").

Otherwise, software that shows currently playing media will show the following string:

![image](https://github.com/vixalien/decibels/assets/37999241/f387dabe-7cb2-4aef-bc8c-5e53ab5145ed)

Reply:
good catch!